src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -1039,86 +1039,90 @@ private void PrepareProjectExecutables()
                 throw new InvalidOperationException("A project resource is missing required metadata"); // Should never happen.
             }
 
-            int replicas = project.GetReplicaCount();
+            var replicas = project.GetReplicaCount();
 
-            var ers = ExecutableReplicaSet.Create(GetObjectNameForResource(project), replicas, "dotnet");
-            var exeSpec = ers.Spec.Template.Spec;
-            exeSpec.WorkingDirectory = Path.GetDirectoryName(projectMetadata.ProjectPath);
+            for (var i = 0; i < replicas; i++)
```

Comment:
Lets not implement this in the PR. We can block replicas for now. I don't think we should aim to fix it in this PR.

Reply:
Could you tell what the specific problems are with switching from replica sets to executables? It's something you and @mitchdenny raised during a meeting. I think you said it would be too difficult to do in .NET 9. I didn't run into any troubles when changing code, and TestShop appeared to still work when I tested it.

The reason I'm trying out executables is I found a fair number of DCP bugs when scaling replica sets during start/stop.

Reply:
I would love us to do this as a follow up. This PR is already doing a lot and changing how replicas work as part of it just increases the complexity of the review and feature (this affect more than start/stop). I would start by not supporting start/stop if there are any replicas and then we can make this change in isolation to see if there are any issues with it.

Reply:
I'm finding a fair number of work specific to scaling replica sets.
* The DCP needs work to make replica sets have stable names between starting and stopping.
* Scaling up replica sets doesn't appear to work in DCP right now (bug?)
* The dashboard or host needs extra code to associate the resource command from the replica instance to the replica set object.

The point I see here is we will spend a lot of time getting replica sets with start/stop/restart, when we could switch now and focus on getting executables working instead.

Reply:
OK if this is easier then go for it. If it starts to snowball then we back it out. I just want to avoid blocking this feature on replicas when it's not important. That's phase 2. Phase 1 is all of the other bootstrapping code. Just disable commands on replicas and do a follow up PR with these changes. 

I'll make it easier to review as well.